src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1092,7 +1110,7 @@ private void onDecodeUnsubscribe(
                 {
                     break;
                 }
-                final int topicKey = System.identityHashCode(topic.filter().asString().intern());
+                final int topicKey = topicKey(topic.filter().asString());
```

Comment:
Does `UNSUBSCRIBE` need a protocol check for `topicFilter == null`, like `SUBSCRIBE`?